package.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "something-fishy",
+  "version": "1.0.0",
```

```suggestion
  "version": "0.0.0",
```
Since this is pre-release, let's set the version to `0.0.0`. We are going to be following Semantic Versioning (aka: [SemVer](https://semver.org/)) so we won't want to change to `1.0.0` until we have an official release. We will talk more about this in the future.